apps/public/views.py

```
@@ -1,8 +1,16 @@
 # apps/public/views.py
 
+"""
+Views for the Public app.
+
+This module contains views for the public-facing pages of the Scopewatch project.
+"""
+
 from django.http import HttpResponse
 from django.shortcuts import render, get_object_or_404
```

Ruff (0.8.2)

`django.shortcuts.get_object_or_404` imported but unused

Remove unused import: `django.shortcuts.get_object_or_404`

(F401)